nix/deku.nix

```diff
@@ -37,7 +37,7 @@ in ocamlPackages.buildDunePackage rec {
   nativeBuildInputs = [ nodejs removeReferencesTo ] ++ npmPackages
     ++ (with ocamlPackages; [ utop reason ]);
 
-  propagatedBuildInputs = with ocamlPackages;
+  buildInputs = with ocamlPackages;
```

Comment:
These packages doesn't need to be "propagated" to consumers so we can just have them as `buildInputs` this makes the closure much smaller and we don't have to use weird tricks to make it small.